content/applications/marketing/marketing_automation/getting_started/testing_running.rst

```diff
@@ -0,0 +1,42 @@
+Testing & running a campaign
+============================
+
+Odoo gives users the ability to test marketing campaigns (and mailings) before officially running
+them.
+
+First, open the :guilabel:`Marketing Automation` application and click on the campaign. Make sure
+the campaign already has activities configured on it. To start a test, click the :guilabel:`Launch a
+Test` button at the top of the template form.
+
+.. image:: testing_running/launch-test.png
+   :align: center
+   :alt: Launch a test button in Odoo Marketing Automation.
+
+When clicked, a pop-up appears, in which the user can choose a specific record to run the test on,
+or the user can create a brand new record to test.
+
+.. image:: testing_running/launch-test-popup.png
```

Comment:
Not sure this screenshot is needed?

Page as a whole is more image-heavy than text...
Wdyt @mivu-odoo ?

Reply:
I'll remove it right away. Thanks for pointing that out.